### PR TITLE
TO-DO LIST APP [STRUCTURE]

### PR DESCRIPTION
**Pull request by BenjaminSemah**
In this project, I have worked on the following;
-built the structure of the to-do list app using webpack.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
## Hi @BenjaminSemah, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

## You have submitted a project, that does not have a descriptive README. Please consider adding a descriptive README to your repository so that users can find information about your project there.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by BenjaminSemah** [comment]
I have updated the README.md file.

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+h1 {
+  text-align: center;
+}
+
+.card {
+  width: 95%;
+  margin: 0 auto;
+  max-width: 400px;
+
+  /* padding: 0 5%; */
```
Comment (Mihndim2020) [inline src/style.css:10]: - [ ]  Please kindly remove commented code from your `code base` as it makes your `code base` dirty and less readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Your project is running as expected.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (BenjaminSemah) [reply src/style.css:10]: [RESOLVED] Commented Code has been removed.

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+![Preview](./preview.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+
```
[the remark this answers is not in the document]
Comment (BenjaminSemah) [reply README.md:17]: [RESOLVED] Steps to clone and run the To Do List app project has been included in README.md file.

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi Team** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**